content/docs/setup-and-administration/getting-access-tokens-using-client-secret/index.md

```diff
@@ -13,19 +13,16 @@ weight: 13
 toc: true
 ---
 
-Having completed the steps at [Application Registration](../configuring-azure-clients-for-non-interactive-use/), and taken note of the `Application ID` and the secret, a token can be obtained with the following command. 
+Having completed the steps at [App Registration](../configuring-appregistrations-for-non-interactive-use/), and taken note of the `CLIENT ID` and the `SECRET`, a token can be obtained with the following command.
 
-Replace `${API_APP_ID}` with the application id, and `${API_APP_SECRET}` with your secret from the application registration; `${FQDN}` is the FQDN of the RKVST SaaS. 
-
-`${TENANT}` is your directory id, see [how to locate your Tenant here](../registering-your-azure-active-directory-with-rkvst/#finding-your-tenant-id).
+Replace `${CLIENTID}` with the application id, and `${SECRET}` with your secret from the application registration; `${FQDN}` is the FQDN of the RKVST SaaS.
 
 ```bash
 $ RESPONSE=$(curl \
-    https://login.microsoftonline.com/${TENANT}/oauth2/token\
+    https://${FQDN}/iam/v1/appidp/token \
```

Comment:
This is likely to be fixed as 'app.rkvst.io/archivist' now. we should just say so.

Reply:
url updated and tested